pulp_container/app/urls.py

```diff
@@ -27,9 +27,9 @@
 router.register(r"^v2/(?P<path>.+)/manifests", Manifests, basename="manifests")
 
 urlpatterns = [
-    url(r"^token/$", BearerTokenView.as_view()),
-    url(r"^v2/$", VersionView.as_view()),
-    url(r"^v2/_catalog", CatalogView.as_view()),
-    url(r"^v2/(?P<path>.+)/tags/list", TagsListView.as_view()),
-    url(r"", include(router.urls)),
+    path("token/$", BearerTokenView.as_view()),
+    path("v2/$", VersionView.as_view()),
+    path("v2/_catalog", CatalogView.as_view()),
+    path("v2/(?P<path>.+)/tags/list", TagsListView.as_view()),
```

Comment:
Looks like the pattern format changed too:
https://docs.djangoproject.com/en/3.2/ref/urls/#path